### PR TITLE
List properties

### PR DESCRIPTION
**Pull request by zfregin**
Review changes to model and resources required for /properties GET endpoint according to data required for properties list page (page 14 of mockup)

### CODE REVIEW

File: db.py [hunk 1]
```diff
@@ -1,2 +1,19 @@
 from flask_sqlalchemy import SQLAlchemy
+# from models.user import UserModel
+# from models.property import PropertyModel
+# from models.revoked_tokens import RevokedTokensModel
 db = SQLAlchemy()
+
+# def seedData():
```
Comment (Chris-Boe) [inline db.py:7]: Should we just delete all these commented lines? Or replace these with the ones added in app.py?

File: models/property.py [hunk 2]
```diff
@@ -9,15 +10,21 @@ class PropertyModel(db.Model):
     city = db.Column(db.String(50))
     state = db.Column(db.String(50))
     zipcode = db.Column(db.String(20))
+    propertyManager = db.Column(db.Integer(), db.ForeignKey('users.id'))
+    tenants = db.Column(db.Integer())
+    dateAdded = db.Column(db.String(50))
```
Comment (Chris-Boe) [inline models/property.py:15]: We may want to consider using the DateTime data type for this dateAdded column

**Review by Chris-Boe — APPROVED** [review]

**Comment by Chris-Boe** [comment]
Looks great, just had the two questions!
Created an issue for this so it's separately tracked from the List Properties Page issue

File: db.py (hunk 1 above)
Comment (htharker42) [reply db.py:7]: yeah, db.py should just be the two lines of code

**Review by htharker42 — APPROVED** [review]
Looks good!

**Pull request merged**